spec/models/exception_hunter/error_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+module ExceptionHunter
+  describe Error do
+    subject { build(:error) }
+
+    describe 'validations' do
+      it { is_expected.to validate_presence_of(:class_name) }
+    end
+
+    describe 'before validation callback' do
+      before { create(:error, ocurred_at: nil) }
+
+      context 'when creating' do
+        it 'sets timestamp' do
+          expect(Error.first.ocurred_at).not_to be_nil
+        end
+      end
+
+      context 'when updating' do
+        let(:error)       { Error.first }
```

Comment:
Not a fan of aligning these, it's easy to miss and it creates inconsistencies

Reply:
could argue at this but no alignment needed when there is just one  🤷‍♀